b2sdk/scan/exception.py

```diff
@@ -76,8 +76,10 @@ def __str__(self):
 
 
 @contextmanager
-def check_invalid_argument(parameter_name: str, message: str,
-                           *exceptions: Type[Exception]) -> Iterator[None]:
+def check_invalid_argument(
+    parameter_name: str, message: str,
+    *exceptions: Type[Exception]
+) -> Iterator[None]:
```

Comment:
I think if we are going multi line, we should use one argument per line - otherwise you might think `exceptions` is the second argument if you are looking up&down and not left&right